examples/dreambooth/train_dreambooth_lora_sdxl.py

```diff
@@ -1132,8 +1240,22 @@ def compute_embeddings(prompt, text_encoders, tokenizers):
                     f" {args.validation_prompt}."
                 )
                 # create pipeline
+                if not args.train_text_encoder:
+                    text_encoder_one = text_encoder_cls_one.from_pretrained(
+                        args.pretrained_model_name_or_path, subfolder="text_encoder", revision=args.revision
+                    )
+                    text_encoder_two = text_encoder_cls_two.from_pretrained(
+                        args.pretrained_model_name_or_path, subfolder="text_encoder_2", revision=args.revision
+                    )
                 pipeline = DiffusionPipeline.from_pretrained(
                     args.pretrained_model_name_or_path,
+                    vae=vae,
+                    text_encoder=accelerator.unwrap_model(text_encoder_one)
+                    if args.train_text_encoder
+                    else text_encoder_one,
+                    text_encoder_2=accelerator.unwrap_model(text_encoder_two)
+                    if args.train_text_encoder
+                    else text_encoder_two,
```

Comment:
```suggestion
                    text_encoder=accelerator.unwrap_model(text_encoder_one),
                    text_encoder_2=accelerator.unwrap_model(text_encoder_two),
```

Reply:
This is not what we want when not training the text encoder actually. 

When not training the text encoder, we precompute the text embeddings and delete the text encoder from memory to save memory. 

This is why during the final inference, we load the text encoders again (only when text encoder is not being traind). 

Makes sense?

Reply:
It's just to make the code more compact - if text_encoder_one is not a `nn.DataParallel` `accelerator.unwrap_model(...)` is simply a no-op. 

So this suggestion is really just a nit to make the code more readable